# Copyright 2020 Benbuck Nason
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.

diff --git i/CMakeLists.txt w/CMakeLists.txt
index 36a6cb5..9428da5 100644
--- i/CMakeLists.txt
+++ w/CMakeLists.txt
@@ -1,2 +1,2 @@
 set(CMAKE_LEGACY_CYGWIN_WIN32 0)
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.10)
@@ -54,7 +54,6 @@ if (ENABLE_CUSTOM_COMPILER_FLAGS)
             /GS
             /Za
             /sdl
-            /W4
             /wd4001
             /D_CRT_SECURE_NO_WARNINGS
         )
